backend/database_handler/models.py

```diff
@@ -0,0 +1,84 @@
+# This file was initially generated with `sqlacodegen postgresql://postgres:postgres@localhost/genlayer_state > models.py`
+from typing import Optional
+
+from sqlalchemy import (
+    BigInteger,
+    CheckConstraint,
+    DateTime,
+    Integer,
+    Numeric,
+    PrimaryKeyConstraint,
+    String,
+    text,
+)
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column
+import datetime
+import decimal
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class CurrentState(Base):
+    __tablename__ = "current_state"
+    __table_args__ = (PrimaryKeyConstraint("id", name="current_state_pkey"),)
+
+    id: Mapped[str] = mapped_column(String(255), primary_key=True)
+    data: Mapped[dict] = mapped_column(JSONB)
+    updated_at: Mapped[Optional[datetime.datetime]] = mapped_column(
+        DateTime(True), server_default=text("CURRENT_TIMESTAMP")
+    )
+
+
+class Transactions(Base):
```

Comment:
@AgustinRamiroDiaz how is the status going to be created?

Reply:
I'm planning to add the `status` field here, and then generate the automatic migration files. That would be the new way of handling schema changes
WDYT?

Reply:
Great, is that going to be added in this PR?

Reply:
Yes! I'm working on it